tabcmd/commands/auth/session.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
         self.token_name = None
         self.token_value = None
         self.password_file = None
+        self.token_file = None
```

Comment:
GitHub doesn't seem to allow me to leave comments on lines that haven't changed, but line 23-24 and 26-27 look duplicated. Is that some Python magic I'm not familiar with?

```python
    TOKEN_CRED_TYPE = "token"
    PASSWORD_CRED_TYPE = "password"

    TOKEN_CRED_TYPE = "token"
    PASSWORD_CRED_TYPE = "password"
```

Reply:
nope, that's just unnecessary duplication. Fixed.